### PR TITLE
Adding perfmetrics directory

### PR DESCRIPTION
**Pull request by etiksha**
This directory contains the necessary files, folders and scripts to run fio tests, extract fio metrics and vm metrics and dump the data to google sheet, all through kokoro.
Starting point: perfmetrics/kokoro/gcp_ubuntu/kokoro_build.sh

### CODE REVIEW

File: perfmetrics/vmmetrics/vmmetrics.py [hunk 1]
```diff
@@ -0,0 +1,234 @@
+"""
+To execute the script:
+>>blaze run vmmetrics {instance} {start time in epoch sec} {end time in epoch sec} {mean period in sec}
```
Comment (avidullu) [inline perfmetrics/vmmetrics/vmmetrics.py:3]: blaze? does this need update?

File: perfmetrics/kokoro/gcp_ubuntu/kokoro_build.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+sudo apt-get update
+echo Installing fio
+sudo apt-get install fio -y
+echo Installing gcsfuse
+curl -L -O https://github.com/GoogleCloudPlatform/gcsfuse/releases/download/v0.41.1/gcsfuse_0.41.1_amd64.deb
+sudo dpkg --install gcsfuse_0.41.1_amd64.deb
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/perfmetrics"
+echo Mounting gcs bucket
+mkdir gcs
+gcsfuse gcs-fuse-dashboard-fio gcs
```
Comment (avidullu) [inline perfmetrics/kokoro/gcp_ubuntu/kokoro_build.sh:12]: shoul use --implicit-dirs --max-conns-per-host 100 --disable-http2 flags

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#i!/bin/bash
+set -e
+sudo apt-get update
+echo Installing fio
+sudo apt-get install fio -y
+echo Installing gcsfuse
+curl -L -O https://github.com/GoogleCloudPlatform/gcsfuse/releases/download/v0.41.1/gcsfuse_0.41.1_amd64.deb
```
Comment (avidullu) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:7]: please make the GCSFuse version a named string and use it everywhere. Will make updating the version easier

GCSFUSE_VERSION=v0.41.1

download/$GCSFUSE_VERSION/..

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+sudo dpkg --install gcsfuse_0.41.1_amd64.deb
+cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse/perfmetrics/scripts"
```
Comment (avidullu) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:9]: does this need to be KOKORO?

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+echo Mounting gcs bucket
+mkdir gcs
```
Comment (avidullu) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:11]: mkdir -p

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+gcsfuse --implicit-dirs --max-conns-per-host 100 --disable-http2 gcs-fuse-dashboard-fio gcs
```
Comment (avidullu) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:12]: pls have these flags as a named string
GCSFUSE_FLAGS="--implicit-dirs --disable-http2 ...
BUCKET_NAME=gcs-fuse-dashboard-fio

gcsfuse $GCSFUSE_FLAGS $GCSFUSE_BUCKET $MOUNT_POINT

Comment (avidullu) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:12]: Can we record the output of this and return error if the mount fails?

**Review by avidullu — COMMENTED** [review]
Looks good overall. Some nit picks to have better visibility

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh (hunk 4 above)
Comment (etiksha) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh:9]: yes

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh (hunk 3 above)
Comment (etiksha) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh]: done

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh (hunk 5 above)
Comment (etiksha) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh]: done

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh (hunk 6 above)
Comment (etiksha) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh]: done

File: perfmetrics/scripts/requirements.txt [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+google-cloud-monitoring
+grpcio
+googleapis.common.protos
+google.auth
+requests
+protobuf==4.21.1
+google.api_core
+google.cloud==0.34.0
+google_api_python_client
+
```
Comment (avidullu) [inline perfmetrics/scripts/requirements.txt:13]: should fio be also here? Or its always preinstalled?

**Pull request merged**